_maps/map_files/SeleneStation/SeleneStation.dmm

```diff
@@ -83743,13 +83743,13 @@
 /turf/open/floor/iron/solarpanel,
 /area/maintenance/solars/port/aft)
 "vTI" = (
-/obj/item/kirbyplants/random,
 /obj/effect/turf_decal/tile/neutral{
 	dir = 8
 	},
 /obj/effect/turf_decal/tile/neutral{
 	dir = 1
 	},
+/obj/machinery/vending/autodrobe,
```

Comment:
```suggestion
/obj/machinery/vending/autodrobe/all_access,
```
This is what you want instead

Reply:
Also why is the branch name helio when this is a change on selene...

Reply:
Because helio and selene are the same maps